batched/dense/impl/KokkosBatched_Gesv_Impl.hpp

```diff
@@ -291,6 +291,8 @@ KOKKOS_INLINE_FUNCTION int TeamVectorStaticPivoting<MemberType>::invoke(
   for (size_t i = 0; i < n; ++i) {
     int row_index, col_index;
     reducer_value_type value;
+    value.val = Kokkos::reduction_identity<value_type>::max();
+    value.loc = Kokkos::reduction_identity<int>::min();
```

Comment:
While it does not change the results (if I understand correctly), it might make sense to replace:
```
value.val = Kokkos::reduction_identity<value_type>::max();
```
by
```
value.val = Kokkos::reduction_identity<value_type>::min();
```
as a reader might misunderstand `if (tmp_v_1(j) > update.val) {` and might think that this will never be true as `value.val = (...)::max()`.

Reply:
I see your point, I chose the current value because of the choice made [here](https://github.com/kokkos/kokkos/blob/715a6ffd53db387b25c6c408dd98c57790c58176/core/src/Kokkos_Parallel_Reduce.hpp#L504) in Kokkos Core.
Maybe we can ask them why they chose the max and not the min?
In any case since the values are reset in the `init()` operator I do not mind using the min

Reply:
I agree that it does not change the results; I just thought that for a different reader in the future (or for us) it might make more sense.
Alternatively, we can just add a comment to say that it will not impact the results even if it could be counterintuitive.